internal/util/auxutil.go

```diff
@@ -0,0 +1,88 @@
+package util
+
+import (
+	"slices"
+
+	etcdaenixiov1alpha1 "github.com/aenix-io/etcd-operator/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EtcdClusterIsReady returns true is condition "Ready" has status equals to "True", otherwise false.
```

Comment:
// EtcdClusterIsReady returns true **if** condition "Ready" has status **equal** to "True", otherwise false.

Is this correct comment?